Mono.Addins/Mono.Addins/ExtensionContext.cs

```diff
@@ -181,7 +184,21 @@ public void RegisterCondition (string id, Type type)
 				ot.Changed -= new EventHandler (OnConditionChanged);
 			info.CondType = type;
 		}
-		
+
+		/// <summary>
+		/// Registers a new localizer in the extension context.
+		/// </summary>
+		/// <param name="id">
+		/// Identifier of the localizer.
+		/// </param>
+		/// <param name="type">
+		/// Localizer type.
+		/// </param>
+		public void RegisterLocalizer (string id, Type type)
```

Comment:
This is not necessary. There is a method for registering conditions because conditions can change depending on the context, but localizer types are global.

Reply:
Ok, it is required by AddinEngine, let's keep it then.